tool/since.dart

```diff
@@ -11,14 +11,16 @@ import 'package:yaml/yaml.dart';
 
 import 'crawl.dart';
 
+// Flip to (re)generate since/linter.yaml contents.
+var regenerateLinterYaml = false;
+
 void main() async {
-// Uncomment to (re)generate since/linter.yaml contents.
```

Comment:
Let's not remove this?

Reply:
I don't understand. Is it better to leave it to bit-rot (like the comments starting on line 28), than to guard with a condition as I have done?

Reply:
I'm neutral TBH.  If this were production code I'd have more of an opinion but since it's more of a helper script I'm OK with it being rough.  For me the comments and structure preserve a bit of context that I'd personally find handy (maybe) when I came back to evolve this script to answer the next question about canonical lints.

That said, if you want me to make sense of this stuff and tighten it up, I'm totally happy to take a pass!